index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;500&family=Poppins:wght@400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Personal Portfolio Website</title>
+</head>
+<body>
+    <nav>
+        <div id="header">
+            <div class="mobile-header">
+                 <div id="primary-logo">
+                     <div id="mobile-logo">
+                     </div>
+                     <div id="logo"></div>
+                 </div>
+                 <div id="mobile-menu">
+                     <!-- <img src="images/ic_menu.png" id="menu-icon" alt="menu-icon" > -->
+                 </div>
+             </div>
+    </nav>
+    <nav>
+             <div class="main-header">
+                 <div class="main-menu">
+                     <ul>
+                         <li><a href="#"> Home</a></li>
+                         <li><a href="#">Portfolio</a></li>
+                         <li><a href="#">About</a></li>
+                         <li><a href="#">Contact</a></li>
+                     </ul>
+                 </div>
+             </div>
+            </div>
+         </div>
+    </nav>
+    <header>
+        <div class="headline">
+            <div>
+             <div id="primary-text">
+                 <h1>Hi👋, I'm Cephas Tokoli</h1>
+             </div>
+             <div class="content">
+                 <p>
+                     Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work. 
+                 </p>
+                 <p> If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                 <button class="button">Start Collaboration</button>
+             </div>
+            </div>
+             <div>
+                 <div class="geometry-image2">
+                     <!-- <img src="images/image_geometry_2.svg" alt="geometry-image"> -->
```

Comment:
Kindly I suggest you remove the commented code from your project to make it more professional and follow HTML best practices. 😄